pset-iterators.js

```diff
@@ -66,6 +71,18 @@ console.log('Problem 2:')
 
 // Add your code below this line
 
+const presidentsNamedJames = presidents.filter(function(singularPresident) {
+  return singularPresident.president.split(" ")[0] === "James"
+})
+
+const getFullName = presidentsNamedJames.map((singularPresident) => {
+  return singularPresident.president
+})
```

Comment:
I liked that you continued the iteration to get a clean list of first names (versus an array of objects)!